react/src/components/Chip.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { BsX } from 'react-icons/bs';
+import styled from 'styled-components/macro';
+import { Button, Flex, Typography } from './index';
+
+export const ChipContainer = styled(props => <Flex {...props} />)`
```

Comment:
I _think_ in styled-components, if you just do `styled(Flex)`, the props should get passed through as Flex is used as the component rather than the anonymous one defined here?

Reply:
When I changed it to only `styled(Flex)`, I got this error: 
<img width="571" alt="Screen Shot 2022-02-18 at 1 49 36 PM" src="https://user-images.githubusercontent.com/30064991/154765931-f904b87a-22c9-4f53-aee6-1adce5eaf53e.png">

I think `styled(Flex)` only works when you're not using the props of `Flex`.